OpenConsole.sln

```diff
@@ -1108,6 +1113,25 @@ Global
 		{CA5CAD1A-D7EC-4107-B7C6-79CB77AE2907}.Release|x64.Build.0 = Release|x64
 		{CA5CAD1A-D7EC-4107-B7C6-79CB77AE2907}.Release|x86.ActiveCfg = Release|Win32
 		{CA5CAD1A-D7EC-4107-B7C6-79CB77AE2907}.Release|x86.Build.0 = Release|Win32
+		{F2ED628A-DB22-446F-A081-4CC845B51A2B}.AuditMode|Any CPU.ActiveCfg = Release|Win32

```

Comment:
@miniksa @DHowett Okay, how did I mess this up?  I thought this was how you un-include a project from audit mode? I really don't think I had anything to do with [this error](https://dev.azure.com/ms/Terminal/_build/results?buildId=83368&view=logs&j=6e3e7df0-e6f0-5f0c-c8d6-63e192049bc0&t=8bda1629-048a-52eb-ac14-ee49f5e51c8a&l=1215)

Reply:
Wait, maybe the U8U16Test error isn't from this PR - I just checked out master and that can't build it in release either, but I've filed #6220 for this